### ISSUE TITLE
Changing accession in plot does not remove the plot data from the accession page

### ISSUE THREAD

**Issue by ch728** [solgenomics/sgn#4513]
I swapped the accession in a plot because there was an error in planting. However,  on the accession's info page I still see the trial name under Experimental Usage and data from that plot in Traits Assayed. This accession should now have no data associated with it.

**Comment by ch728** [comment]
So I think I have more insight.  The initial accession in the plot was 19MOS0205  and I swapped it with 192BN0205 . Plot level data and plant level data were collected for this plot. Changing the accession in the plot removed the plot-level data from 19MOS0205, but the plant-level data remains associated with it. To make matters worse, when I download the data for 192BN0205  for that trial it only downloads the plot-level data and not the plant-level data. The plant-level data is not associated with 192BN0205 even though the plot, and thus plants in the plot, was updated to 192BN0205. Updating the accession in a plot should alter the linkages with the plant-level and tissue-level data too, not just the plot-level data.

**Comment by lukasmueller** [comment]
Yes, it should. But you may have to update some of the materialized views to see it in all instances on the website.

cheers
Lukas
________________________________

**Comment by ch728** [comment]
Yeah, I tried that by refreshing the wizard. It is not just about the web page view, it effects the data downloaded. I also noticed that is not just the phenotypes. Changing the accession in the plot does not change the accession for the plants in the plot. If I download plant data it shows the old accession in the download instead of the new one.

**Comment by lukasmueller** [comment]
There is another matview that needs to be refreshed I think. Is there any way to access this instance?

**Comment by ch728** [comment]
Unfortunately, not right now. I am currently running my breedbase instance on a personal server. I fixed the problem for now by deleting the data and trial, and then re-uploading everything.  If I see this problem in the future should I try running this script https://github.com/solgenomics/sgn/blob/master/bin/refresh_matviews.pl with "fullview" to force a refresh?

**Comment by lukasmueller** [comment]
yes, that's the one.

Let me know if you think we need a brief phone call to discuss some of the issues you encountered.

thanks and cheers
Lukas

________________________________

**Comment by ch728** [comment]
There is still some weirdness going on here. I will do some more testing and see If I can up with a simple reproducible example.
